src/forms/StaticField.tsx

```diff
@@ -12,11 +13,13 @@ interface StaticFieldProps {
 export function StaticField(props: StaticFieldProps) {
   const { label, value, children } = props;
   const tid = useTestIds(props, defaultTestId(label));
+  const id = useId();
```

Comment:
Curious why the `id`?

Reply:
For accessibility purposes `<label>` elements should be tied to a value via a `for` -> `id` pairing.

Reply:
Ah yeah, that makes sense, I completely missed the `htmlFor={id}` part of the diff. Cool!